catalogs/catalog-jdbc-common/src/main/java/com/datastrato/gravitino/catalog/jdbc/operation/JdbcDatabaseOperations.java

```diff
@@ -77,6 +77,12 @@ public List<String> listDatabases() {
     }
   }
 
+  protected ResultSet getDatabase(Connection connection, String databaseName) throws SQLException {
```

Comment:
could this be reused by PG? PG still use system tables.

Reply:
Fixed